source/_integrations/yandextts.markdown

```diff
@@ -9,17 +9,24 @@ ha_domain: yandextts
 ha_platforms:
   - tts
 ha_integration_type: integration
+related:
+  - docs: /docs/configuration/
+    title: Configuration file
+related:
+  - docs: /docs/configuration/
+    title: Configuration file
```

Comment:
Double trouble 😛
```suggestion
```

Reply:
Thank you for the catch, @klaasnicolaas :+1: